iree/turbine/kernel/wave/utils/classes.py

```diff
@@ -13,3 +14,15 @@ class KernelLaunchInfo:
     shared_memory_bytes: int = 0
     func_name: str = ""
     grid_str: str = ""
+
+
+############################################
+# Wave Ops related Utils
+############################################
+
+# GPU shuffle modes
+class ShuffleMode(Enum):
```

Comment:
nit: can we have the same order as in handler?
idx, xor, up, down or this order there.

Reply:
ahh nice catch! :)

Reply:
switched it up to the enum ordering of gpu dialect in MLIR which is xor, down, up, idx! :)

Reply:
https://github.com/llvm/llvm-project/blob/8ba3a232d1985a75cf771f8832e911b6f8abe86b/mlir/include/mlir/Dialect/GPU/IR/GPUOps.td#L1284-L1287